### DISCUSSION TITLE
Onboard JST EH connector

### DISCUSSION THREAD

**Discussion by jsapede** [revk/xESP32-Faikout#267] [Q&A]
Hello all
Juste recieved my faikin and tried to connect it using a s21 cable from brp069b41 8 had for a while.

But when i openened the interior unit i found a s21 référence but only with a JST eh connector (see photo) on the motherboard

![PXL_20240316_125733691.jpg](https://github.com/revk/ESP32-Faikin/assets/66303110/87a86835-9e02-47c4-910f-d7066c079134)

The interior unit is atxf50a2v1e. Will th Daikin work if i put a double jst eh wiring harness between the motherboard and the faikin ?

**Comment by revk** [comment]
That looks like a normal S21 plug, so yes, 1 to 1 JST cable from that to Faikin.
